cwl/echo.cwl

```diff
@@ -1,21 +1,66 @@
-arguments:
-- $(inputs.message)
-baseCommand:
-- echo
-- -n
+#!/usr/bin/env cwl-runner
+
 class: CommandLineTool
 cwlVersion: v1.0
+
+label: Print given message using `echo` and capture output
+
+doc: >
+  The `echo` command-line utility is used to print a message,
+  which is captured from standard output.
+
+  Visit the Linux manual page for more information:
+    https://www.freebsd.org/cgi/man.cgi?query=echo
+
+requirements:
+  - class: ShellCommandRequirement
+
 hints:
   DockerRequirement:
     dockerPull: sagebionetworks/dockstore-tool-template:0.0.4
+
+baseCommand:
+  - echo
+
 inputs:
+
+  extra_args:
+    type: string?
+    label: Additional command-line (usually optional) arguments
+    inputBinding:
+      position: 0
+      shellQuote: false
+
   message:
     type: string
+    inputBinding:
+      position: 1
+
 outputs:
-  out:
+
+  output_file:
+    type: File
+    format: edam:format_1964  # Plain text
```

Comment:
What is the default value? Is this necessary to set on an example that is intended to be as simple as possible?

Reply:
It's not strictly required. If the intent of this repo is to be as simple as possible, then this can be omitted. If the intent is to demonstrate best practices, then this should arguably be included. 

I don't know what the historical intent of this repo was, so I defer to others in this matter. I'm fine with reverting this line addition.

Reply:
Oh, and to answer your first question, I don't think there's a default value.